tests/pmodel/test_pmodel_global_array.py

```diff
@@ -1,6 +1,9 @@
 """Runs tests of the P Model using global arrays as inputs.
 
 This is used to benchmark the pyrealm predictions against the rpmodel implementation.
+
+TODO - this test is currently broken because there is an issue with calculating the
```

Comment:
Should this then be removed, rather than leaving something outdated and broken on here?